verify-docker-versions

```diff
@@ -8,14 +8,24 @@ VERSION_MINOR_MIN_DOCKER_ENGINE="11"
 VERSION_MINOR_MIN_DOCKER_COMPOSE="6"
 
 compare_versions() {
-  VERSION_ACTUAL=$1
-  VERSION_MINIMUM=$2
+  VERSION_MAJOR=$1
+  VERSION_ACTUAL=$2
+  VERSION_MINIMUM=$3
 
   if [[ -z $VERSION_ACTUAL ]]; then
     echo "ERROR: Actual version is empty (hint: is it installed, or version format change?)."
     return 1
   fi
 
+  if [ "$VERSION_MAJOR" == "2" ]; then
+    return 0
+  fi
+
+  if [ "VERSION_MAJOR" != "1" ]; then
```

Comment:
```suggestion
  if [ "$VERSION_MAJOR" != "1" ]; then
```

Reply:
:see_no_evil: I tried going with the local flow here, I usually like the `{}` syntax for this reason.
Apologies